### PR TITLE
Broaden the scope of 2s

### PR DESCRIPTION
**Pull request by Rouxles**
Probably needs rewording

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2n1) Disputes are only permitted during the competition, within 30 minutes after the disputed incident happened and before the start of any following rounds of the relevant event.
     - 2n2) The WCA Delegate must resolve the dispute before the start of the next round of the event.
     - 2n3) The competitor must accept all final rulings of the WCA Delegate. Penalty: disqualification from the competition.
-- 2s) Competitors with disabilities that may prevent them from abiding by one or more WCA Regulations may request special accommodations from the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate at least two weeks before the competition.
+- 2s) Competitors under certain circumstances (e.g. disability, religious beliefs) that may prevent them from abiding by one or more WCA Regulations may request special accommodations from the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate at least two weeks before the competition.
```
Comment (Rouxles) [inline wca-regulations.md]:
```suggestion
- 2s) Competitors under certain circumstances (e.g. disability) that may prevent them from abiding by one or more WCA Regulations may request special accommodations from the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate at least two weeks before the competition.
```

**Review by Nevseros — APPROVED** [review]
It seems it covers all possible cases now, LGTM!

File: wca-regulations.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2n1) Disputes are only permitted during the competition, within 30 minutes after the disputed incident happened and before the start of any following rounds of the relevant event.
     - 2n2) The WCA Delegate must resolve the dispute before the start of the next round of the event.
     - 2n3) The competitor must accept all final rulings of the WCA Delegate. Penalty: disqualification from the competition.
-- 2s) Competitors with disabilities that may prevent them from abiding by one or more WCA Regulations may request special accommodations from the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate at least two weeks before the competition.
+- 2s) Competitors under certain circumstances (e.g. disability) that may prevent them from abiding by one or more WCA Regulations may request special accommodations from the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate at least two weeks before the competition.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
- 2s) Competitors with circumstances (e.g. disability) that may prevent them from abiding by one or more WCA Regulations may request special accommodations from the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate at least two weeks before the competition.
```
'With' still seems more natural to me. I'd welcome the input of native US English speakers.

Comment (Rouxles) [reply wca-regulations.md]: No yeah this reads way better to me too

Comment (Rouxles) [reply wca-regulations.md]: edited your comment slightly - removed certain, which i think reads better now

**Pull request merged**